packages/apps/registry-backend/.env.example

```diff
@@ -4,3 +4,5 @@ MONGO_DB_NAME=testing
 IS_DEVELOPMENT=true
 CORS_ALLOWED_DOMAIN=http://localhost:3000
 DEBUG=express:*
+EXPECTED_AUDIENCE=localhost:3000
```

Comment:
when i started the registry backend it complained that these things were missing so i added them to the .env.example